ksqldb-functional-tests/src/test/resources/query-validation-tests/joins.json

```diff
@@ -1966,6 +1966,26 @@
       "outputs": [
         {"topic": "OUTPUT", "key": "user_0", "value": {"IMPRESSION_ID": 24, "URL": "urlA"}, "timestamp":  12}
       ]
+    },
+    {
+      "name": "streams with no key columns",
```

Comment:
dont want to be too picky, but let's also add a stream no-key --> table join

Reply:
done